fix hardcoded text elements in the plots of the forecast class (#769)
CLIMADA-project/climada_python: CHANGELOG.md

- `util.lines_polys_handler` solve polygon disaggregation issue in metre-based projection [#666](https://github.com/CLIMADA-project/climada_python/pull/666)
- Problem with `pyproj.CRS` as `Impact` attribute, [#706](https://github.com/CLIMADA-project/climada_python/issues/706). Now CRS is always stored as `str` in WKT format.
- Correctly handle assertion errors in `Centroids.values_from_vector_files` and fix the associated test [#768](https://github.com/CLIMADA-project/climada_python/pull/768/)

> - fixed hardcoded text elements in the `Forecast` class [#769](https://github.com/CLIMADA-project/climada_python/issues/769)

Reviewer: ```suggestion
- Text in `Forecast` class plots can now be adjusted [#769](https://github.com/CLIMADA-project/climada_python/issues/769)
```
